flink-python/pyflink/datastream/functions.py

```diff
@@ -19,6 +19,7 @@
 import abc
 from typing import Union, Any, Dict
 
+from apache_beam import TimeDomain
```

Comment:
I think we should import Flink's TimeDomain

Reply:
Yes, it was an error import.